tools/clang/test/SemaHLSL/invalid-rovbuffer-elements.hlsl

```diff
@@ -0,0 +1,31 @@
+// RUN: %dxc -Tlib_6_3 -verify %s
+
+// RasterizerOrderedBuffer has suficiently different errors that it requires its own verifiction test.
```

Comment:
I think this is no longer the case, right?  Can this be rolled back into the other test then?

Reply:
No, this is still the case. The issue is the absence of `RWRasterizerOrderedBuffer` and the different behavior regarding no template arguments. Essentially it's a RW buffer without the RW prefix. It only affects the first few lines, but I value those lines as I didn't find any tests for them elsewhere and I couldn't come up with macro skullduggery to cram this round peg into that square hole. I do enjoy macro skullduggery, but I think it has bothered reviewers in the past where I might have more clearly just created a new test with some redundancy as I've done here.